vllm/engine/metrics.py

```diff
@@ -331,6 +331,15 @@ def log(self, stats: Stats) -> None:
                 generation_throughput=generation_throughput)
 
             # Log to stdout.
+            logger_data = {
+                "avg_prompt_throughput": prompt_throughput,
+                "avg_generation_throughput": generation_throughput,
+                "running": stats.num_running,
+                "swapped": stats.num_swapped,
+                "pending": stats.num_waiting,
+                "gpu_kv_cache_usage": stats.gpu_cache_usage * 100,
+                "cpu_kv_cache_usage": stats.cpu_cache_usage * 100
+            }
```

Comment:
Sorted and `event` added:

```suggestion
            logger_data = {
                "avg_generation_throughput": generation_throughput,
                "avg_prompt_throughput": prompt_throughput,
                "cpu_kv_cache_usage": stats.cpu_cache_usage * 100
                "event": "engine-metrics",
                "gpu_kv_cache_usage": stats.gpu_cache_usage * 100,
                "pending": stats.num_waiting,
                "running": stats.num_running,
                "swapped": stats.num_swapped,
            }
```

Reply:
This is a scenario where I think we DEFINITELY want `logger_data`